scripts/rebuild_chains.py

```
@@ -1,126 +1,71 @@
-    "anmix", "tokenrouter", "tokenlb", "pooled",  # chinese gateways
-    "navy",  # chinese gateway
-    "zenllm-free",  # free tier
-    "futureppo",  # daily checkin
-    "kilocode", "kilo",  # free tier
-}
+from chain_policy import candidate_key, blocked_reason, load_policy, sanitize_chain
 
 
-def load_working_models(path="/tmp/opencode/working_models.json"):
-    """Load {provider: [(model, n, ttft, tps), ...]} from JSON dump.
+DEFAULT_CONFIG = Path("/tmp/opencode/virtual_models.yaml")
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/opencode/virtual_models.yaml"

(S108)